### DISCUSSION TITLE
V1.4 Release Date ?

### DISCUSSION THREAD

**Discussion by sshearer** [dialogic-godot/dialogic#671] [General]
Hi, it's great to see the activity for the solution and with the v1.4 Change Log info, it makes me wonder:

1. Is there a target release date being considered for v1.4, and available for download in the godot Asset Library ?

2. I know there's a long wish list but is there any particular focus or overall goal for v1.4 ?

**Comment by Jowan-Spooner** [comment]
Hi, this was asked a long time ago, but I'm going to answer it anyways; better late then never. 
1. v1.4 is planned to be released relatively soon, in the beginning of february 2022.

2. There was/is no overall goal for 1.4, our work is not consistent and organized enough for that yet. It consists mostly of things we care about or bugs that need fixing.
